pipelines/azure-pipelines.docker.yml

```diff
@@ -10,15 +10,31 @@ pool:
   vmImage: 'ubuntu-latest'
 
 variables:
-- group: JMETER_AZURE_PRINCIPAL
 - group: JMETER_TERRAFORM_SETTINGS
 
 steps:
-- script: |
-    az login --service-principal --username $(ARM_CLIENT_ID) --password $(ARM_CLIENT_SECRET) --tenant $(ARM_TENANT_ID)
-    az account set --subscription $(ARM_SUBSCRIPTION_ID)
-  displayName: 'Login on Azure CLI'
 
-- script: |
-    az acr build -t $(TF_VAR_JMETER_DOCKER_IMAGE) -r $(TF_VAR_JMETER_IMAGE_REGISTRY_NAME) -f $(Build.SourcesDirectory)/docker/Dockerfile .
+- task: AzureKeyVault@1
```

Comment:
Hi @hepsi204!
I think we can remove these 2 first tasks:
```yaml
- task: AzureKeyVault@1
  inputs:
    azureSubscription: $(TF_VAR_SERVICE_CONNECTION_NAME)
    KeyVaultName: $(TF_VAR_KEY_VAULT_NAME)
    SecretsFilter: 'acr-secret,arm-client-secret'
  displayName: 'Get secrets from Azure Key Vault'

- task: AzureCLI@2
  displayName: 'ACR login'
  inputs:
    azureSubscription: $(TF_VAR_SERVICE_CONNECTION_NAME)
    scriptType: bash
    scriptLocation: inlineScript
    inlineScript: |
      az acr login -n $(TF_VAR_JMETER_IMAGE_REGISTRY_NAME) -u $(TF_VAR_JMETER_IMAGE_REGISTRY_USERNAME) -p $(acr-secret)
```

The `az acr login` is used to allow native `Docker` commands to used with `docker` CLI. In our case, the last task can build the image without a login. 

Can you please remove them?
Good work so far :)

Reply:
done 🙂

Reply:
Thanks!